### ISSUE TITLE
UI shows negative transferrable balance

### ISSUE THREAD

**Issue by ggwpez** [opensquare-network/subsquare#4635]
Account `13TRAXTALwNp5vApqwiE74fg8G8ypMyaF9TxRfs4RwrCwxUE` shows negative -6.4 transferrable balance. Not sure what this should mean.

<img width="1130" alt="Screenshot 2024-09-01 at 15 36 49" src="https://github.com/user-attachments/assets/adec154c-f874-412c-bbb6-32e8de5b0dcb">

For comparison what PJS shows:

<img width="465" alt="Screenshot 2024-09-01 at 15 38 01" src="https://github.com/user-attachments/assets/d0e80631-795e-4df5-9b1e-dc99f9cf6f83">

**Comment by wliyongfeng** [comment]
Our calculation `transferrable = free - frozen`, but it seems incorrect.


```javascript
export const accountTransferrableBalanceSelector = createSelector(
  accountInfoSelector,
  (info) => {
    if (!info) {
      return 0;
    }

    const { free, frozen } = info;
    return new BigNumber(free || 0).minus(frozen || 0).toNumber();
  },
);
```

https://github.com/opensquare-network/subsquare/blob/main/packages/next-common/store/reducers/myOnChainData/account.js#L38-L48

**Comment by ggwpez** [comment]
PAPI uses this: https://github.com/polkadot-js/api/pull/5957 
>This uses the formula: free - max(maybeEd, frozen - reserve)
Where `maybeEd` means if there is no frozen and reserves it will be zero, else it will be the existential deposit.

**Comment by wliyongfeng** [comment]
Yes, I've fixed it(#4637) according to [the wiki page](https://wiki.polkadot.network/docs/learn-account-balances). We will release it soon.
